cmd/crosschainbank.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+)
+
+// crosschainBankCmd represents the command to get balances across chains
+func crosschainBankCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "crosschainbank",
+		Aliases: []string{"cb"},
+		Short:   "query about balances across chains",
+	}
+
+	cmd.AddCommand(
+		getEnabledChainbalancesCmd(),
+	)
+
+	return cmd
+}
+
+func getEnabledChainbalancesCmd() *cobra.Command {
```

Comment:
Reposting from TG: I don't think that having the default be just a list of balances is ideal. Would be nice to break the results out by chain (i.e. `map[chain-name][]string` with the strings being rendered coins from the account) and have the feature for a top level balance view (i.e. `map[denom]sdk.Int` or `[]string` rendered) behind a feature flag.